### PR TITLE
add initial pull request template for github

### PR DESCRIPTION
**Pull request by mahibi**
Signed-off-by: Marcel Hibbe <dev@mhibbe.de>

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+### 🚧 TODO
+
+- [ ] ...
+
+### 🏁 Checklist
+
+- [ ] ⛑️ Tests (unit and/or integration) are included or not needed
+- [ ] 🔖 Capability is checked or not needed 
+- [ ] 🔙 Backport requests are created or not needed: `/backport to stable-xx.x`
+- [ ] 📅 Milestone is set
```
Comment (nickvergessen) [inline .github/pull_request_template.md:17]: IIRC people outside of the Org can not set milestones, so not sure this makes a lot of sense.

**Review by nickvergessen — APPROVED** [review]

File: .github/pull_request_template.md (hunk 1 above)
Comment (mahibi) [reply .github/pull_request_template.md:17]: i would say not all of the points from the checklist must be checked by community members.
so for backport requests and milestone i suggest to add "(should be done by nextcloud employees)" ?

Comment (AndyScherzinger) [reply .github/pull_request_template.md:17]: Best to add an not rendered comment about the optional items on the list, same is true for screenshots

Comment (nickvergessen) [reply .github/pull_request_template.md:17]: But then people think they can skip it although they should just do 2 screenshots.
Let's keep it and the milestone for now and see if there is community feedback afterwards

Comment (AndyScherzinger) [reply .github/pull_request_template.md:17]: Makes sense! Release early, release often :)

**Review by AndyScherzinger — APPROVED** [review]

**Pull request merged**